src/diagnostics/LogPointsManager.ts

```diff
@@ -92,9 +93,12 @@ class DebugSessionManager {
 
         const result: ISetLogpointResponse = await this._debugSession.customRequest("setLogpoint", { scriptId, lineNumber, columnNumber, expression });
         if (result.error !== undefined && result.error !== null) {
-            vscode.window.showErrorMessage("Cannot set logpoint successfully. Please refer to DEBUG CONSOLE for details.");
+            vscode.window.showErrorMessage("Cannot set logpoint successfully. Please refer to DEBUG CONSOLE and https://aka.ms/logpoints for details.");
```

Comment:
Should we use a more specific link?

Reply:
Will change to `https://aka.ms/logpoints#setting-logpoints`